data/STIAANWOL.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Stiaan Wolfaardt",
+  "type": "personal",
+  "bio": "Software Engineer based in Cape Town, South Africa.",
+  "avatar": "https://github.com/STIAANWOL.png",
```

Comment:
Hi @STIAANWOL,  thank you for contributing  :tada: 
You can delete this line, it is not required anymore :+1: 
```Suggestion
```